WapoJS/src/service.rs

```diff
@@ -218,9 +263,37 @@ impl Service {
     }
 
     pub fn run_default_module(&self) -> Result<js::Value> {
-        let default_fn = self.context().get_global_object().get_property("module")?.get_property("exports").unwrap_or_default();
+        let ctx = self.context();
+        let default_fn = ctx.get_global_object().get_property("module")?.get_property("exports").unwrap_or_default();
         if default_fn.is_function() {
-            return self.call_function(default_fn, ());
+            let ret = self.call_function(default_fn, ());
+
+            //
+            // Auto set promise result to scriptOutput
+            //
+            if let Ok(val) = ret {
```

Comment:
Can we do this complex logic in JS?
for example define the following code in JS:
```js
Wapo.callModuleEntry = async function callModuleEntry() {
    const entry = globalThis.module?.exports;
    if (typeof entry=== 'function') {
         try {
               globalThis.scriptOutput = await entry();
          } catch (error) {
               globalThis.scriptError = error;
          }
    }
}
```
Then in the Rust we just do this:
```rust
let entry_fn = ctx.get_global_object().get_property("Wapo")?.get_property("callModuleEntry")?;
if entry_fn.is_function() {
    return self.call_function(entry_fn, ());
}
```

Reply:
During my testing, I found that it became more difficult to obtain results from `isolateEval`. Therefore, I prefer to keep the current implementation.